modules/installation-guide/partials/proc_run-several-workspaces-at-a-time.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// advanced-configuration
+
+[id="run-several-workspaces-at-a-time_{context}"]
+= Running more than one workspace at a time
+
+This procedure describes how to run more than one workspace simultaneously, allowing a user to perform several tasks at once.
+
+.Prerequisites
+
+* The `{platforms-cli}` tool is available.
+* An instance of {prod-short} running in {platforms-name}.
++
+[NOTE]
+====
+The following commands use the default {platforms-namespace}, `{prod-namespace}`, as a user's example for the `-n` option.
+====
+
+.Procedure
+
+. Remove the limit of concurrently running workspaces per user:
```

Comment:
While technically true, you're actually setting a limit of -1, rather than removing the limit. :D So...

> Change the default limit of `1` to `-1` to allow an unlimited number of concurrent workspaces per user:

Reply:
Done